source/DSMainWindow.cpp

```diff
@@ -98,6 +102,36 @@ DSMainWindow::DSMainWindow(QWidget* parent):
     setWindowTitle("Despeect");
 }
 
+void DSMainWindow::createActionLoadVoice() {
+    const QIcon openIcon = QIcon::fromTheme("document-open", QIcon(":/images/open.png"));
+
+    QAction *openAct = new QAction(openIcon, QObject::tr("Load Configuration File"), this);
+    openAct->setShortcuts(QKeySequence::Open);
+    openAct->setStatusTip(tr("Load Configuration File"));
+    QObject::connect(openAct,
```

Comment:
Come prima, le connect le raggrupperei tutte e le separerei dal resto del codice. Non riconosco la sintassi [=], è una lambda expression? Fico, syntax candy, però io piuttosto il segnale lanciato da QAction::triggered lo aggancerei ad uno slot dentro DSMainWindow (dove mettere le istruzioni che hai scritto nel blocco della lambda), mi sembra più leggibile (più semplice, perchè è leggibile anche così) così il codice, più manutenibile ed organizzato

Reply:
Si è una lambda-expression
per la sintassi[=]:(da http://en.cppreference.com/w/cpp/language/lambda)

[=] captures all automatic variables used in the body of the lambda
 by copy and current object by reference if exists

ovvero cattura tutte le variabili automatiche per valore (e il this per riferimento?)